.github/workflows/_lints.yml

```diff
@@ -0,0 +1,77 @@
+name: Reusable workflow for lints
+
+on:
+  workflow_call:
+    inputs:
+      target:
+        required: true
+        type: string
+
+jobs:
+  clippy:
+    name: Clippy (MSRV)
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ${{ inputs.target }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install build dependencies
+        run: sudo apt install libudev-dev
+      - name: Run Clippy
+        uses: auguwu/clippy-action@1.1.2
+        with:
```

Comment:
The `deny` etc. options are currently broken in the action. I've opened an issue upstream: https://github.com/auguwu/clippy-action/issues/13

The reason my workflow worked before is because `args` was being completely ignored. I've opened an issue for that upstream as well: https://github.com/auguwu/clippy-action/issues/14

So currently there is no way to configure anything except `--all-features`. I'll comment out the `deny` lines for now, and leave `--all-targets` in the ignored `args` input, and then once an update to the action is available, we can re-enable these.